treeherder/model/error_summary.py

```diff
@@ -214,11 +214,17 @@ def get_error_search_term_and_path(error_line):
     tokens = error_line.split(" | ")
     search_term = None
     path_end = None
+    is_crash = False
 
     if len(tokens) >= 3:
+        if 'PROCESS-CRASH' in tokens[0]:
```

Comment:
Please drop line 217 and replace 220+221 with `is_crash = 'PROCESS-CRASH' in tokens[0]`